.github/workflows/canary.yml

```diff
@@ -0,0 +1,63 @@
+name: canary
+on:
+  - pull_request
+  - push
+jobs:
+  main:
+    name: '${{matrix.node}} on ${{matrix.os}}'
+    runs-on: ${{matrix.os}}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dcodeIO/setup-node-nvm@master
+        with:
+          node-version: ${{matrix.node}}
+      - run: npm install -g npm
+      - run: npm install
+      - run: npm run build
+
+      - name: checkout mdast-util-from-markdown
+        uses: actions/checkout@v2
+        with:
+          repository: syntax-tree/mdast-util-from-markdown
+          path: canary/mdast-util-from-markdown
+      - name: test mdast-util-from-markdown
+        run: |
+          npm install
+          npx rimraf "**/micromark"
+          npx rimraf "**/micromark-util-*"
```

Comment:
this removes the pinned versions of these libraries that npm installed, meaning node will continue searching upward, until it finds the parent `micromark/node_modules` folder with the just-built version of micromark.